Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```diff
@@ -418,7 +418,7 @@
 - type: entity
   parent: DrinkBottleGlassBaseFull
   id: DrinkAleBottleFull
-  name: Magm-Ale
```

Comment:
This sounds like it was intended to be a proper noun, and proper nouns should remain capitalized.